ci/release/update-version.sh

```
@@ -57,7 +57,10 @@ if [[ -z "${SKIP_MD_UPDATE}" ]]; then
    sed_runner "s|https:\/\/docs.nvidia.com\/nemo\/agent-toolkit\/\([0-9|\.]\+\)|https:\/\/docs.nvidia.com\/nemo\/agent-toolkit\/${NEXT_SHORT_TAG}|g" src/nat/meta/pypi.md
 fi
 
-
+NAT_NOTEBOOKS=($(find ./examples/notebooks -name "*.ipynb" | sort ))
```

Shellcheck (0.11.0)

[warning] Prefer mapfile or read -a to split command output (or quote to avoid splitting).

(SC2207)